pkg/router/template/router.go

```diff
@@ -291,6 +291,9 @@ func secretToPem(secPath, outName string) error {
 	if err != nil {
 		return err
 	}
+	if pemBlock[len(pemBlock) - 1] != byte('\n') {
```

Comment:
Should we protect against the case where len(pemBlock) is 0?

Reply:
Indeed!  Thanks!  Fixed.